src/components/base/JIcon/JIcon.js

```diff
@@ -9,19 +10,26 @@ type Props = {
   name: string,
   size: JIconSize,
   color: JIconColor,
+  isCurrentColor: boolean,
 }
 
+const files = require.context('../../../public/assets/icons/new-pack', true, /.*\.svg$/)
+files.keys().forEach(x => files(x))
```

Comment:
Looks like black magic is done here. Could you please describe what is happening?